packages/machine-extractor/src/MachineExtractResult.ts

```diff
@@ -2846,6 +2846,13 @@ function getImplementationObject(
       const propKey = getObjectPropertyKey(node);
       const val = fileContent.slice(node.value.start!, node.value.end!);
       out[propKey] = val;
+    } else if (t.isObjectMethod(node)) {
+      const propKey = getObjectPropertyKey(node);
```

Comment:
It would produce wrong (or at least quite weird) output at that function expression~ that you create a couple of lines below this. This particular action would serialize to smth like this:
```ts
function fetch user() {}
```

And this isn't a valid JS ;p

Reply:
Done. It might be better to first transform AST and then extract them, but that's an idea for future.